src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -382,4 +386,14 @@ public DatabaseService getDatabaseService(final String databaseName) {
     public ExaOperation getExaOperation() {
         return this.exaOperation;
     }
-}
+
+    /**
+     * Get the time zone.
+     *
+     * @return time zone
+     */
+    public TimeZone getTimeZone() {
+        assert (this.timeZone != null);
```

Comment:
I don't like asserts. Can you throw an exception instead?

Reply:
This is not the right place anyway. Moved check to `clusterConfigurationIsAvailable()` with regular exception.